### DISCUSSION TITLE
04/04

### DISCUSSION THREAD

**Discussion by revature-ives** [revature-ives/PickandPayStore#17] [Daly Spring]
**What I dId yesterday**
@Costin79 created a proposal categories view
@mattparker412  created a proposal Product view details
@sfrazier851 wrote a document whit the different categories of products and assets for the app
@algoives created proposal track order view

**Obstacles**
@Costin79 margin problems whit GitHub
@algoives  understand the tracking logic 

**What I going to do** 
@Costin79 design products view
@mattparker412 add reviews to the product detail view
@sfrazier851 continue working in data base adding actual products to sell
@algoives define the logic of tracking orders

**Comment by Costin79** [comment]
Added pictures to the project. 3 categories: droids, ground vehicles, spaceships. 7 pictures(products) in each category
.
